src/web/components/StickyAd.tsx

```diff
@@ -6,11 +6,12 @@ import { namedAdSlotParameters } from '@root/src/model/advertisement';
 
 type Props = {
     name: AdSlotType;
-    height: number;
+    height?: number;
 };
-const adSlotWrapper = (height: number) => css`
+const adSlotWrapper = ({ height }: { height?: number }) => css`
     position: static;
-    height: ${height}px;
+    min-height: 274px;
```

Comment:
to mimic frontend, I set a min-width instead of a width. This avoids having spacing between the right column ad and the most read column

Reply:
Note: local testing shows that this doesn't prevent with larger ad blocks from rendering